tool/crawl.dart

```diff
@@ -188,7 +191,11 @@ Future<List<String>> _fetchSdkTags() {
   return github.repositories
       .listTags(slug)
       .map((t) => t.name)
+      .where((t) => t != null)
       .where((t) {
+        if (t == null) {
```

Comment:
It seems like this test and the test on line 194 are doing the same thing, but maybe I'm missing something.

Reply:
Indeed, thanks. The `.where((t) => t != null)` idea didn't pan out (since I still get a stream of nullable Strings) but I forgot to remove it.  👍

Reply:
(There's no `whereType<T>()` on streams?)